.github/workflows/python-tests.yml

```diff
@@ -220,10 +220,10 @@ jobs:
           ref: ${{ inputs.ref }}
           persist-credentials: false
           submodules: "recursive"
-      - name: Set up Python 3.8
+      - name: Set up Python 3.11
```

Comment:
nit: why `3.11` and not `3.12`?

Reply:
Snowpark does not yet support Python 3.12: https://github.com/snowflakedb/snowpark-python/blob/b96234e717540ca8f06147afc7a9e27505065022/setup.py#L27